azext_iot/tests/test_iot_central_int.py

```diff
@@ -187,10 +187,14 @@ def test_central_device_registration_info_filter_unassociated(self):
         device_info_results = []
         json_result = result.get_output_in_json()
         for device in json_result:
-            device_info_results.append(device.get("device_info"))
+            device_info_results.append(device.get("device_registration_info"))
 
         for device in device_info_results:
-            assert device.get("deviceStatus") == device_status_expected
+            assert device.get("device_status") == device_status_expected
```

Comment:
nice! one final note is that it would be good to have the check itself (i.e. `device.get("display_name") == some_value`) rather than just the truthiness check

Reply:
maybe even a `len(device) == 5`

Reply:
to ensure we aren't adding more props

Reply:
I'm not sure if this actually got resolved, did you mean to `wontfix` this